utils/firestore.js

```diff
@@ -392,6 +392,40 @@ const retrieveParticipantsEligibleForIncentives = async (siteCode, roundType, is
     }
 }
 
+const removeDocumentFromCollection = async (connectID) => {
+    try {
+        // Collections with document related to participant
+        const collectionArray = [
```

Comment:
would prefer we reference an existing constant of survey collection names such as `moduleConceptsToCollections` from `shared.js`

if not, this will just be one more spot in code we need to remember to update when a new survey is added

Reply:
Solved